src/test/Calculator.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Renderer from 'react-test-renderer';
+import Calculator from '../components/Calculator';
+
+it('renders correctly Calculator Component ', () => {
+  const tree = Renderer
+    .create(<Calculator />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Following the project requirements, you should use also React Testing Library to simulate user interaction. Check out this [link ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/testing_react_with_react_testing_library.md)to learn how to do that.